lib/util/template/index.ts

```diff
@@ -24,6 +24,9 @@ handlebars.registerHelper('containsString', (str, subStr) =>
   str?.includes(subStr)
 );
 
+// eslint-disable-next-line eqeqeq
+handlebars.registerHelper('equals', (arg1, arg2) => arg1 == arg2);
```

Comment:
What is the significance of the double equals instead of triple equals here? e.g. you want the ability for `0 == false`?

Reply:
Exactly, to leave some wiggle room for type coercion. It's probably a bad idea, but I figured it might make sense for number/booleans & number/string pairs. I also had the idea of adding a `strictEquals` for `===`.

Personally I'd be happy to change this to `===`

Reply:
I'd prefer this one (equals) uses ===

Reply:
Done!